pom.xml

```diff
@@ -54,6 +54,9 @@
   </distributionManagement>
 
   <properties>
+    <revision>4.50</revision>
+    <changelist>-SNAPSHOT</changelist>
```

Comment:
Fine but why not enable CD then? The only reason to hold off was the fear of regressions from incrementalification, not CD per se.

Reply:
I am not a fan of the version numbers used in JEP-229 releases.

Reply:
I should have mentioned that I am not a fan of JEP-229 more generally. The labeling scheme and automatically generated change logs have long been a source of frustration for me. If I have the choice between using JEP-229 and avoiding it, I will choose to avoid it.